### PR TITLE
Re-include variable definitions and prepare for last round

### PR DESCRIPTION
**Pull request by coroa**
Changes to variabledefs are in the second commit:
https://github.com/IAMconsortium/concordia/pull/59/commits/ab9b424223f2468f742e67304ad556adf3edb922

### CODE REVIEW

File: data/variabledefs-rescue.csv [hunk 1]
```diff
@@ -39,11 +39,12 @@ CO2,Solvents Production and Application,Mt CO2/yr,TRUE,country,anthro_CO2.nc,CO2
 CO2,Transportation Sector,Mt CO2/yr,TRUE,country,anthro_CO2.nc,CO2_em_anthro,TRA
 CO2,Waste,Mt CO2/yr,TRUE,country,anthro_CO2.nc,CO2_em_anthro,WST
 CO2,CDR Afforestation,Mt CO2/yr,TRUE,global,CDR_CO2.nc,CO2_em_anthro,A/R
-CO2,CDR BECCS,Mt CO2/yr,TRUE,global,CDR_CO2.nc,CO2_em_anthro,BECCS
+CO2,CDR BECCS,Mt CO2/yr,TRUE,region,CDR_CO2.nc,CO2_em_anthro,BECCS
 CO2,CDR DACCS,Mt CO2/yr,TRUE,country,CDR_CO2.nc,CO2_em_anthro,DAC_CDR
 CO2,CDR EW,Mt CO2/yr,TRUE,country,,,
 CO2,CDR Industry,Mt CO2/yr,TRUE,country,CDR_CO2.nc,CO2_em_anthro,IND_CDR
 CO2,CDR OAE Uptake Ocean,Mt CO2/yr,TRUE,region,CDR_CO2.nc,CO2_em_anthro,OAE_CDR
+CO2,Ocean Alkalinity,,FALSE,region,CDR_CO2.nc,CO2_em_anthro,OAE_CDR
```
Comment (coroa) [inline data/variabledefs-rescue.csv:47]: Should this gridded ocean alkalinity be part of the CO2_em_anthro file? What is the unit?

Comment (gidden) [reply data/variabledefs-rescue.csv:47]: per our discussion:

- variable name -> 'Alkalinity Addition'
- yes to the co2 anthro file (no better place atm)
- unit should come from @merfort and be provided as a flux

File: data/variabledefs-rescue.csv [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+CO2,International Shipping,Mt CO2/yr,TRUE,global,shipping_CO2.nc,CO2_em_anthro,SHP
+CO2,Residential Commercial Other,Mt CO2/yr,TRUE,country,anthro_CO2.nc,CO2_em_anthro,RCO
+CO2,Solvents Production and Application,Mt CO2/yr,TRUE,country,anthro_CO2.nc,CO2_em_anthro,SLV
+CO2,Transportation Sector,Mt CO2/yr,TRUE,country,anthro_CO2.nc,CO2_em_anthro,TRA
+CO2,Waste,Mt CO2/yr,TRUE,country,anthro_CO2.nc,CO2_em_anthro,WST
+CO2,CDR Afforestation,Mt CO2/yr,TRUE,global,CDR_CO2.nc,CO2_em_anthro,A/R
+CO2,CDR BECCS,Mt CO2/yr,TRUE,region,CDR_CO2.nc,CO2_em_anthro,BECCS
+CO2,CDR DACCS,Mt CO2/yr,TRUE,country,CDR_CO2.nc,CO2_em_anthro,DAC_CDR
+CO2,CDR EW,Mt CO2/yr,TRUE,country,,,
+CO2,CDR Industry,Mt CO2/yr,TRUE,country,CDR_CO2.nc,CO2_em_anthro,IND_CDR
```
Comment (gidden) [inline data/variabledefs-rescue.csv:45]: shouldn't this be the same spatial proxy as daccs?

File: data/variabledefs-rescue.csv [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+CO2,Industrial Sector,Mt CO2/yr,TRUE,country,anthro_CO2.nc,CO2_em_anthro,IND
+CO2,International Shipping,Mt CO2/yr,TRUE,global,shipping_CO2.nc,CO2_em_anthro,SHP
+CO2,Residential Commercial Other,Mt CO2/yr,TRUE,country,anthro_CO2.nc,CO2_em_anthro,RCO
+CO2,Solvents Production and Application,Mt CO2/yr,TRUE,country,anthro_CO2.nc,CO2_em_anthro,SLV
+CO2,Transportation Sector,Mt CO2/yr,TRUE,country,anthro_CO2.nc,CO2_em_anthro,TRA
+CO2,Waste,Mt CO2/yr,TRUE,country,anthro_CO2.nc,CO2_em_anthro,WST
+CO2,CDR Afforestation,Mt CO2/yr,TRUE,global,CDR_CO2.nc,CO2_em_anthro,A/R
+CO2,CDR BECCS,Mt CO2/yr,TRUE,region,CDR_CO2.nc,CO2_em_anthro,BECCS
+CO2,CDR DACCS,Mt CO2/yr,TRUE,country,CDR_CO2.nc,CO2_em_anthro,DAC_CDR
+CO2,CDR EW,Mt CO2/yr,TRUE,country,,,
```
[the remark this answers is not in the document]
Comment (gidden) [reply data/variabledefs-rescue.csv:44]: In fact, maybe we can do that already directly? i.e., take current agricultural cells from LUH2 / harmonization year from @pascal-sauer's dataset.

File: data/variabledefs-rescue.csv [hunk 4]
```diff
@@ -0,0 +1,136 @@
+gas,sector,unit,include_in_total,griddinglevel,proxy_path,proxy_name,proxy_sector
```
Comment (gidden) [inline data/variabledefs-rescue.csv:1]: I know this is from early days, but should we call this `proxy_name` or rather `output_file`? I've lost a bit the thread which of the columns are related to reading in the data versus writing it out.

**Comment by gidden** [comment]
Thanks @coroa - I went through the csv and added all comments directly there

File: data/variabledefs-rescue.csv [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+CH4,Forest Burning,Mt CH4/yr,TRUE,country,openburning_CH4.nc,CH4_em_openburning,FRTB
+CH4,Grassland Burning,Mt CH4/yr,TRUE,country,openburning_CH4.nc,CH4_em_openburning,GRSB
+CH4,Industrial Sector,Mt CH4/yr,TRUE,country,anthro_CH4.nc,CH4_em_anthro,IND
+CH4,International Shipping,Mt CH4/yr,TRUE,global,shipping_CH4.nc,CH4_em_anthro,SHP
+CH4,Peat Burning,Mt CH4/yr,TRUE,country,openburning_CH4.nc,CH4_em_openburning,PEAT
+CH4,Residential Commercial Other,Mt CH4/yr,TRUE,country,anthro_CH4.nc,CH4_em_anthro,RCO
+CH4,Solvents Production and Application,Mt CH4/yr,TRUE,country,anthro_CH4.nc,CH4_em_anthro,SLV
+CH4,Transportation Sector,Mt CH4/yr,TRUE,country,anthro_CH4.nc,CH4_em_anthro,TRA
+CH4,Waste,Mt CH4/yr,TRUE,country,anthro_CH4.nc,CH4_em_anthro,WST
+CO2,Aggregate - Agriculture and LUC,Mt CO2/yr,TRUE,global,,,
```
Comment (gidden) [inline data/variabledefs-rescue.csv:30]: On mattermost, we agreed to rename this variable `Deforestation and other LUC`. It should be treated in the same way as the aforestation variable (same harm/ds/grid approach and pattern)

**Review by coroa — COMMENTED** [review]
Thanks for all the comments, i updated a few things and added comments. Main thing i am unsure about currently is whether to keep A/R as the MAgPIE output we have trialed or to switch to the non-urban landmask.

File: data/variabledefs-rescue.csv (hunk 4 above)
Comment (coroa) [reply data/variabledefs-rescue.csv]: Ok, rewrote to `output_variable`

**Pull request merged**